tests/install-branch.js

```diff
@@ -27,5 +29,23 @@ const takedown = async () => {
   }
 };
 
+const waitForPage = async (page) => {
+  const tries = 20;
+
+  await page.goto(interviewConstants.PETITIONER_URL);
+
+  for (i=0; i<tries; i++) {    
+    const element = await page.$('#daMainQuestion');
+    if (element) {
+      console.log("found question on page");
```

Comment:
Just want to confirm that these statements are intentionally here, and not leftovers from debugging. Otherwise, looks good.

Reply:
Yeah I figured they were nice to have when watching the pipeline running